src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.DiagnosticProvider.cs

```diff
@@ -61,8 +62,7 @@ public abstract class DiagnosticProvider
                     {
                         return await FixAllContextHelper.GetDocumentDiagnosticsToFixAsync(
                             fixAllContext,
-                            fixAllContext.ProgressTracker,
-                            (d, c) => DocumentExtensions.IsGeneratedCode(d, c)).ConfigureAwait(false);
+                            fixAllContext.ProgressTracker).ConfigureAwait(false);
```

Comment:
`IGeneratedCodeDefinitionService` has been moved to the shared layer, so no need to explicitly pass in this delegate.

Reply:
👍